theano/sandbox/cuda/elemwise.py

```diff
@@ -132,9 +132,9 @@ def c_src_kernel(self, node, nodename, nd):
         #TODO: What if the scalar_op needs support_code??
         task_code = self.scalar_op.c_code(
             Apply(self.scalar_op,
-                  [scalar.Scalar(dtype=input.type.dtype)()
+                  [scalar.Scalar(dtype=input.type.dtype).make_variable()
                    for input in node.inputs],
-                  [scalar.Scalar(dtype=output.type.dtype)()
+                  [scalar.Scalar(dtype=output.type.dtype).make_variable()
```

Comment:
This call extract_stack. This call linechache.checkcache() for each stack level for each call to extract_stack.

This call os.stat() on the source files. So this cause problem on clusters when Theano is installed on the network file system like on helios. This cause load of 2k-6k IO access seconds as we take the stack very frequently.